evals/solvers/providers/openai/whisper_solver.py

```diff
@@ -71,3 +77,70 @@ def _transcribe(self, wav_bytes: bytes) -> str:
         file = io.BytesIO(wav_bytes)
         file.name = "test.wav"
         return self.client.audio.transcriptions.create(model="whisper-1", file=file).text
+
+
+# TODO: merge with the code in local_gpu_solver.py
```

Comment:
This is using an older implementation (without batching). If not necessary, I can remove it from this PR and add it later with more polish.

Reply:
I'll defer to you and Peter as to how easy it would be to port over the batched executor to handle this case too.